README.md

```diff
@@ -42,7 +42,11 @@ represent the tiles.
 
 ## A1.1
 
-«replace this with your answer»
+S -> "r" | "b" | "g"
+S -> R S R | G S G | B S B
+R -> "r"
+G -> "g"
+B -> "b"
```

Comment:
No deduction, but there's no need for the quotes: the r, b, and b are already terminals:

~~~
       S -> r | g | b
       S -> r  S  r
       S -> g  S  g
       S -> b  S  b
~~~